script/hassfest/quality_scale.py

```diff
@@ -1299,6 +1300,7 @@ class Rule:
     "daikin",
     "danfoss_air",
     "datadog",
+    "daybetter_light_local",
```

Comment:
Please add a quality scale file instead. New integrations require Bronze tier:

https://developers.home-assistant.io/docs/core/integration-quality-scale/

https://developers.home-assistant.io/docs/core/integration-quality-scale/#keeping-track-of-the-implemented-rules